### PR TITLE
Refactors the display to meet spec #39

### PR DESCRIPTION
**Pull request by azyth**
solves #39

### CODE REVIEW

**Comment by azyth** [comment]
@kyleoliveira ready for a review. code compiles and flashes well for several configurations.

File: lib/Ohmbrewer_Heating_Element.cpp [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #include "Ohmbrewer_Heating_Element.h"
+#include "Ohmbrewer_Screen.h"
```
Comment (kyleoliveira) [inline lib/Ohmbrewer_Heating_Element.cpp]: Any reason you added this given that you didn't change the rest of the file?

File: lib/Ohmbrewer_Pump.cpp [hunk 2]
```diff
@@ -1,4 +1,6 @@
 #include "Ohmbrewer_Pump.h"
+#include "Ohmbrewer_Publisher.h"
+#include "Ohmbrewer_Screen.h"
```
Comment (kyleoliveira) [inline lib/Ohmbrewer_Pump.cpp]: Any reason you added these given that you didn't change the rest of the file?

**Comment by kyleoliveira** [comment]
Looks good to me. I'm gonna say go with it, though you should try adding like 2 Thermostats and 6 temps and seeing what happens.

**Pull request merged**